### PR TITLE
Flink/EP 1.1.4

### PR DESCRIPTION
**Pull request by tagarr**
Modify samples for flink and ep to use tls for flink deployments

### CODE REVIEW

File: event-processing/cr-examples/eventprocessing/kubernetes/README.md [hunk 1]
```diff
@@ -17,6 +17,7 @@ To find out more about the samples see [the documentation](https://ibm.github.io
       - OIDC auth configuration template
       - Customized TLS configuration including, root CA and UI certificate
       - Persistent storage
+      - Connect to a secure flink deployment
```
Comment (vigneshibmpub) [inline event-processing/cr-examples/eventprocessing/kubernetes/README.md:20]:
```suggestion
      - Connect to a secure Flink deployment
```

File: flink/tls-setup/README.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+# TLS setup
+
+This folder contains 3 yaml files
```
Comment (vigneshibmpub) [inline flink/tls-setup/README.md:3]:
```suggestion
This folder contains 3 YAML files:
```

File: flink/tls-setup/README.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
...
+
+- create-cluster-issuer.yaml Creates a CA backed cluster issuer. You need to modify <cert-manager-namespace> to the namespace your cert manager is installed in.
```
Comment (vigneshibmpub) [inline flink/tls-setup/README.md:5]:
```suggestion
- `create-cluster-issuer.yaml`:  Creates a CA-backed cluster issuer. You need to modify `<cert-manager-namespace>` to the namespace where your cert manager operator is installed.
```

File: flink/tls-setup/README.md [hunk 4]
```diff
@@ -0,0 +1,7 @@
...
+- create-flink-operator-jks-certificate.yaml Creates a jks certificate for the flink operator. Change <flink-operator-namespace> to the namespace your flink operator is installed in and <Base64 encoded password> to be a base64 encoded password you want to use
```
Comment (vigneshibmpub) [inline flink/tls-setup/README.md:6]:
```suggestion
- `create-flink-operator-jks-certificate.yaml`: Creates a JKS certificate for the Flink operator. Modify `<flink-operator-namespace>` to the namespace where your flink operator is installed, and `<base64-encoded-password>` to be a Base64-encoded password that you want to use.
```

File: flink/tls-setup/README.md [hunk 5]
```diff
@@ -0,0 +1,7 @@
...
+- create-flink-deployment-jks-certificate Creates a jks certfiicate for your flink deployment. Change <flink-namespace> to the namespace you're creating your flinkdeployment, <flink-deployment-name> to the name of your flink dpeloyment and <Base64 encoded password> to be a base64 encoded password you want to use
```
Comment (vigneshibmpub) [inline flink/tls-setup/README.md:7]:
```suggestion
- `create-flink-deployment-jks-certificate.yaml`: Creates a JKS certfiicate for your Flink deployment. Modify `<flink-namespace>` to the namespace you are creating your `FlinkDeployment` instance, `<flink-deployment-name>` to the name of your Flink deployment, and `<base64-encoded-password>` to be a Base64-encoded password that you want to use.
```

File: flink/tls-setup/create-flink-deployment-jks-certificate.yaml [hunk 6]
```diff
@@ -0,0 +1,32 @@
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: <flink-deployment-name>-cert-password
+  namespace: <flink-namespace>
+type: Opaque
+data:
+  password: <Base64 encoded password>
```
Comment (vigneshibmpub) [inline flink/tls-setup/create-flink-deployment-jks-certificate.yaml:9]:
```suggestion
  password: <base64-encoded-password>
```

File: flink/tls-setup/create-flink-operator-jks-certificate.yaml [hunk 7]
```diff
@@ -0,0 +1,31 @@
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: operator-certificate-password
+  namespace: <flink-operator-namespace>
+type: Opaque
+data:
+  password: <Base64 encoded password>
```
Comment (vigneshibmpub) [inline flink/tls-setup/create-flink-operator-jks-certificate.yaml:9]:
```suggestion
  password: <base64-encoded-password>
```

**Review by vigneshibmpub — COMMENTED** [review]
Added some suggestions.

**Review by dibbles — APPROVED** [review]
LGTM

**Pull request merged**